testsuite/features/support/remote_node.rb

```diff
@@ -55,6 +55,8 @@ def initialize(host, port: 22)
     if (PRIVATE_ADDRESSES.key? host) && !$private_net.nil?
       @private_ip = net_prefix + PRIVATE_ADDRESSES[host]
       @private_interface = 'eth1'
+      _out, _err, code = ssh('ip address show ens4')
+      @private_interface = 'ens4' if code.zero?
```

Comment:
same

Reply:
this code could be made more similar to what we have for the public interface (same file)